lib/magma/server/update.rb

```diff
@@ -55,4 +57,36 @@ def load_revisions
     log(m.complaints)
     @errors.concat(m.complaints)
   end
+
+  def update_any_file_links
+    # Here, if there are any File attributes in the revisions and
+    #   the storage is configured to use Metis instead of AWS,
+    #   we'll send a request to the Metis "copy" route, using the
+    #   Etna::Client.
+    if Magma.instance.config(:storage).fetch(:provider).downcase == 'metis'
+
+      # Etna::Client.new throws an exception if the host config includes protocol,
+      # i.e. https://metis-dev.etna-development.org will throw an exception
+      raise Etna::BadRequest, 'Storage host should not include protocol' if
```

Comment:
This error might be raised when you load storage; the client probably doesn't need to know about this.

Reply:
Is there another exception that I should raise here, to let a deployment person know that storage is configured incorrectly? Or are we just relying on outside knowledge of the acceptable format?